src/main/java/sg/ncl/MainController.java

```diff
@@ -290,7 +289,11 @@ public String loginSubmit(
             } else if (jwtTokenString != null || !jwtTokenString.isEmpty()) {
                 JSONObject tokenObject = new JSONObject(jwtTokenString);
                 String token = tokenObject.getString("token");
-                id = tokenObject.getString("id");
+                String id = tokenObject.getString("id");
+
+                // TODO: get user type from jwt token
+//                String userType = tokenObject.getString("type");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)